lib/client/node.js

```diff
@@ -289,9 +289,16 @@ class NodeClient extends Client {
    * @returns {Promise}
    */
 
-  estimateFee(blocks) {
+  async estimateFee(blocks) {
     assert(blocks == null || typeof blocks === 'number');
-    return this.call('estimate fee', blocks);
+    let query = '/fee';
```

Comment:
This can just be (or similar):
```js
let query = '/fee';
if (blocks != null)
  query += `?blocks=${blocks}`;
return this.get(query);
```

Reply:
thanks, much cleaner

Reply:
Should we consider using the node.js module `querystring` to handle uri encoding?
This case may be simple enough to not need to use the module.

```javascript
if (blocks != null)
  query += `?${qs.encode({blocks}}`;
```

https://nodejs.org/api/querystring.html

Reply:
With `querystring`, it may be possible to write as:
```js
return this.get(`/fee?${qs.encode({blocks})}`);
```
 However, how it reads now is fairly simple.

Reply:
I agree that it reads far more simple now. I think it should stay the way it is.